list2/ex10/index.js

```diff
@@ -0,0 +1,44 @@
+import promptSync from 'prompt-sync';
+const prompt = promptSync();
+
+let sum = 0;
+let count = 0;
+let smallestValue;
+let evenCount = 0;
+
+let exitLoop;
+
+export function numberAnalysis() {
+
+    do {
+        console.log('Inserte an INEGER number.')
+        const userNumbers = parseInt(prompt('>'));
+
+        sum += userNumbers;
+        count++;
+
+        if (count === 1) {
+            smallestValue = userNumbers;
+        } else if (userNumbers < smallestValue) {
+            smallestValue = userNumbers;
+        }
```

Comment:
You are making one thing dependent on the other by checking count here. I would suggest changing this validation to something like:
```suggestion
       if (smallestValue === undefined || userNumbers < smallestValue) {
            smallestValue = userNumbers;
        }
```